### DISCUSSION TITLE
Who is who on the Masterclass

### DISCUSSION THREAD

**Discussion by pvgenuchten** [ejpsoil/soildata-assimilation-guidance#15] [Announcements]
This page provides an option to introduce yourself as a participant to the masterclass on soil data assimilation, which has been organized in januari 2023.

**Comment by pvgenuchten** [comment]
Hello, I'm Paul van Genuchten, I work at [ISRIC - World Soil information](https://www.isric.org) in Wageningen NL as an SDI-specialist. I'm the initial author of a substantial number of the recipes in this WIKI

**Reply by sforstner** [reply]
Hi, my name is Stefan Forstner, I'm a terrestrial ecosystem ecologist by training, with a focus on soil carbon and nitrogen. At the Austrian Research Centre for Forests (BFW) I'm contributing to the Agricultural Soil Map of Austria (https://bodenkarte.at/).

https://orcid.org/0000-0003-2891-1597
https://www.researchgate.net/profile/Stefan-Forstner-2

**Reply by Fatemehbah** [reply]
Hello . I am Fatemeh Bahreini. I work at Agroscope -Switzerland. I am working as a soil science background which is based on modeling and Machin learning.

**Reply by Gengiska** [reply]
Hi! I'm Nicola Riitano, i'm working at the italian national institute for environmental research and protection (ISPRA), in the field of Land Consumption and Land Degradation.

**Reply by marliesdekeizer** [reply]
Hi, I'm Marlies de Keizer, working at Wageningen Environmental Research in the team Applied Spatial Research. I am a data scientist and currently working on transforming soil data from our own format to defined standards. For this training I'm interested in getting to know Hale Studio.

**Reply by sforstner** [reply]
> Hi, my name is Stefan Forstner, I'm a terrestrial ecosystem ecologist by training, with a focus on soil carbon and nitrogen. At the Austrian Research Centre for Forests (BFW) I'm contributing to the Agricultural Soil Map of Austria (https://bodenkarte.at/).
> 
> https://orcid.org/0000-0003-2891-1597 https://www.researchgate.net/profile/Stefan-Forstner-2

Thanks everyone for organising this course. Since the material covered is beyond my field of expertise, I will need time to figure out what is relevant for our organisation & where/if I will dive deeper. This will take time, hence I'm not going to continue with the course right now; but will come back to the material provided eventually, as needed. Thanks again!

**Reply by pvgenuchten** [reply]
> Thanks everyone for organising this course. Since the material covered is beyond my field of expertise, I will need time to figure out what is relevant for our organisation & where/if I will dive deeper. This will take time, hence I'm not going to continue with the course right now; but will come back to the material provided eventually, as needed. Thanks again!

thanx for notifying Stefan, hope you find still some material of interest. What type of material did you expect?

**Reply by geographika** [reply]
Hi all, my name is Seth Girvin (@geographika) and I'm joining the masterclass as a representative of the [MapServer](https://mapserver.org/) project - an opensource platform for publishing spatial data and interactive mapping applications to the web. I'll be giving a short presentation on MapServer tomorrow morning, and will be joining the [https://gitter.im/ejpsoil/soildata-assimilation-guidance](gitter chat) if you have any questions (about software rather than soils - of which my knowledge is somewhat limited!)
